src/main/java/org/gridsuite/explore/server/services/ParametersService.java

```diff
@@ -89,9 +89,8 @@ public void updateParameters(UUID parametersUuid, String parameters, ParametersT
     public UUID createParameters(UUID sourceParametersUuid, ParametersType parametersType) {
         String parametersServerBaseUri = remoteServicesProperties.getServiceUri(genericParametersServices.get(parametersType));
         Objects.requireNonNull(sourceParametersUuid);
-        String urlParams = parametersType == ParametersType.VOLTAGE_INIT_PARAMETERS ? "?" + HEADER_DUPLICATE_FROM + "=" + sourceParametersUuid : "/" + sourceParametersUuid;
         var path = UriComponentsBuilder
-                    .fromPath(DELIMITER + SERVER_API_VERSION + "/parameters" + urlParams)
+                    .fromPath(DELIMITER + SERVER_API_VERSION + "/parameters/" + sourceParametersUuid)
```

Comment:
I see that if we change here with `duplicateFrom` form, we will need other PRs for each computaiton services... let's do it ?

Reply:
other PRs have been created